conn_test.go

```diff
@@ -658,7 +658,7 @@ func TestQueryTimeout(t *testing.T) {
 		if err != ErrTimeoutNoResponse {
 			t.Fatalf("expected to get %v for timeout got %v", ErrTimeoutNoResponse, err)
 		}
-	case <-time.After(10*time.Millisecond + db.cfg.Timeout):
+	case <-time.After(40*time.Millisecond + db.cfg.Timeout):
```

Comment:
any specific reason for 40 over 10?

Reply:
When running the `stress` I go a failure out of 10000 without that. I assume this was happening before as well. I don't think it will make a difference in the travis tests

Reply:
Yea, agree. Just was wondering what's happening that warrants the increase in the waiting there.